docs/sql/data_types/numeric.md

```diff
@@ -11,21 +11,32 @@ The types `UTINYINT`, `USMALLINT`, `UINTEGER`, `UBIGINT` and `UHUGEINT` store wh
-| `INTEGER`   | `INT4`, `INT32`, `INT`, `SIGNED` |                              -2147483648 |                              2147483647 |
-| `BIGINT`    | `INT8`, `INT64` `LONG`           |                     -9223372036854775808 |                     9223372036854775807 |
-| `HUGEINT`   | `INT128`                         | -170141183460469231731687303715884105728 | 170141183460469231731687303715884105727 |
-| `UTINYINT`  | -                                |                                        0 |                                     255 |
-| `USMALLINT` | -                                |                                        0 |                                   65535 |
-| `UINTEGER`  | -                                |                                        0 |                              4294967295 |
-| `UBIGINT`   | -                                |                                        0 |                    18446744073709551615 |
-| `UHUGEINT`  | -                                |                                        0 | 340282366920938463463374607431768211455 |
+| Name        | Aliases                          |                                      Min |                                     Max |     Size&nbsp;in&nbsp;Bytes |
+| :---------- | :------------------------------- | ---------------------------------------: | --------------------------------------: | ----------------: |
+| `TINYINT`   | `INT1`                           |                                     -128 |                                     127 |                 1 |
+| `SMALLINT`  | `INT2`, `INT16` `SHORT`          |                                   -32768 |                                   32767 |                 2 |
+| `INTEGER`   | `INT4`, `INT32`, `INT`, `SIGNED` |                              -2147483648 |                              2147483647 |                 4 |
+| `BIGINT`    | `INT8`, `INT64` `LONG`           |                     -9223372036854775808 |                     9223372036854775807 |                 8 |
+| `HUGEINT`   | `INT128`                         | -170141183460469231731687303715884105728 | 170141183460469231731687303715884105727 |                16 |
+| `UTINYINT`  | -                                |                                        0 |                                     255 |                 1 |
+| `USMALLINT` | -                                |                                        0 |                                   65535 |                 2 |
+| `UINTEGER`  | -                                |                                        0 |                              4294967295 |                 4 |
+| `UBIGINT`   | -                                |                                        0 |                    18446744073709551615 |                 8 |
+| `UHUGEINT`  | -                                |                                        0 | 340282366920938463463374607431768211455 |                16 |
```

Comment:
I guess we could also add VARINT here?

Reply:
I don't think that's a good idea.
The minimum and maximum are both incredibly huge, and the Size in Bytes is variable, which is exactly why I've made a separate section for it.

I do understand why it would fit there, because it *just* a numeric like the others in the list, but it's giant

Reply:
Well, that's a summary, on top of the page, and most likely the only thing that 99.9% of the people will read from it, so that's why I think is important to add it there. The ranges might look "confusing" at first glance, but then readers can check the actual varint description you added, if it's not immediately clear to them.

I think for bytes you can have the range 4 - 2ˆ23 bytes. Min and max can be a value * 10^...

I can also do that in a later PR.